examples/flower.rs

```diff
@@ -0,0 +1,110 @@
+//! Draws a simple geometric sort of flower with customizable dimensions.
```

Comment:
Thank you! I don't know if gorgeous is the right term, but I'll take the compliment :wink:. I left it out of the examples index for now, because even with the maximum speed, it runs *so* slowly. Recording it in this state simply would have been more troublesome than waiting until it performs better. I'll add in when part 2 is complete.